backend/database/user.go

```diff
@@ -43,12 +44,20 @@ func (mDb MongoDatabase) GetUserByEmail(ctx context.Context, email string) (*Use
 	return user, nil
 }
 
+func (mDb MongoDatabase) SetUserRole(ctx context.Context, userID primitive.ObjectID, role string) error {
+	update := bson.M{"$set": bson.M{"role": role}}
+	_, err := db.Collection("users").UpdateByID(ctx, userID, update)
+
+	return err
+}
+
 func (mDb MongoDatabase) GetUserByID(ctx context.Context, userID ObjectID) (*User, error) {
 	user := new(User)
 	filter := bson.M{"_id": userID}
 	err := db.Collection("users").FindOne(ctx, filter).Decode(&user)
 	if err != nil {
 		return nil, err
 	}
+	user.Password = ""
```

Comment:
This wouldn't be needed, if the query was made with a aggregate pipeline, that unsets the passwordfield. It seemed like an overly complex solution, so I opted for this one.

Reply:
Good point, and I agree that it might be a bit overkill for now. The current solution seems more straightforward and fits the need well.